lang/python/python3/patches/021-openssl-deprecated.patch

```diff
@@ -0,0 +1,88 @@
+--- a/Modules/_ssl.c
++++ b/Modules/_ssl.c
+@@ -47,6 +47,7 @@ static PySocketModule_APIObject PySocketModule;
+ 
+ /* Include OpenSSL header files */
+ #include "openssl/rsa.h"
++#include "openssl/dh.h"
```

Comment:
I think this has [already been fixed](https://github.com/python/cpython/blob/v3.8.1/Modules/_ssl.c#L66) (https://github.com/python/cpython/pull/9347).

Reply:
Removed.